cmd/daemon/daemon.go

```diff
@@ -43,7 +43,7 @@ func main() {
 	entryLog.Info("starting hybridnet daemon",
 		"known-features", feature.KnownFeatures(), "commit-id", gitCommit)
 
-	config, err := daemonconfig.ParseFlags()
+	config, err := daemonconfig.ParseFlags(entryLog)
```

Comment:
no need to take logger as an input, we can log error and config outside

Reply:
Yes, I've found that too. Fixed.